WordPress/gutenberg
# Spec Parser: Move generated spec parser to package

This pull request seeks to treat `lib/parser.php` as a generated build artifact of the `block-serialization-spec-parser` package, both removing it from `lib/` and from source altogether, assumed to be generated as part of `npm run build` in time for a package publish. This treatment has been applied to the `parser.js` generated result as well.

**Testing instructions:**

`lib/parser.php` is not actually used by Gutenberg, so it should have no impact on the effective behavior of the plugin.

Verify that `parser.js` and `parser.php` are generated to `packages/block-serialization-spec-parser` after `npm install`.

Ensure that tests pass covering the generated files:

```
npm run test-unit packages/block-serialization-spec-parser
```
